frontend/src/components/LikeButtonPicture/LikeButtonPicture.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../styles/prelude.scss";
+
+.like-component-picture{
+    display: flex;
+    flex-flow: row nowrap;
+    font-size: 15px;
+    position: relative;
```

Comment:
Like the last comment, let me know what you were trying to do with this. I'll try to this of a solution that doesn't rely on a relative positioning and placement